variables.mk

```diff
@@ -6,7 +6,8 @@
 HELP_COMPILATION_VARIABLES = \
 "   JAVA_HEAP_SIZE    = if overridden, set the default java heap size (default is 8G)" \
 "   JAVA_TOOL_OPTIONS = if overridden, set underlying java tool options (default sets misc. sizes and tmp dir)" \
-"   SBT_OPTS          = if overridden, set underlying sbt options (default uses options in .sbtopts)" \
+"   SBT_OPTS          = set additional sbt command line options (these take the form -Dsbt.<option>=<setting>) " \
+"                       See https://www.scala-sbt.org/1.x/docs/Command-Line-Reference.html\#Command+Line+Options" \
```

Comment:
GH shows the comment extending over the `\`. Did you confirm this works with `make help`?

Reply:
yeah